app/code/Magento/CacheInvalidate/README.md

```diff
@@ -1,2 +1,11 @@
-The CacheInvalidate module is used to invalidate the Varnish cache if it is configured.
-It listens for events that request the cache to be flushed or cause the cache to be invalid, then sends Varnish a purge request using cURL.
\ No newline at end of file
+# Magento_CacheInvalidate module
+
+The **Magento_CacheInvalidate** module is used to invalidate the Varnish cache if it is configured.
+
+The **Magento_CacheInvalidate** module listens for events that request the cache to be flushed or cause the cache to be invalid, then sends Varnish a purge request using cURL.
+
+## Extensibility
```

Comment:
Is it really applicable here? I don't see a lot of classes to customize.

Reply:
Done